string rscp3g
string ecio3g
string psc3g
string band3g
